marshmallow/schema.py

```diff
@@ -452,7 +452,7 @@ def accessor(cls, func):
 
     ##### Serialization/Deserialization API #####
 
-    def dump(self, obj, many=None, update_fields=True, **kwargs):
+    def dump(self, obj, many=None, update_fields=True, only=None, **kwargs):
```

Comment:
I'm not sure about adding `only` as a param here. I think it'd be good to limit the number of params to `dump` and `load`. Otherwise, users might expect all `Meta` options (e.g. `only`, `exclude`, `include`) to be parameters.